### ISSUE TITLE
SC 2.5.8 Pointer Target Spacing - Minimum pointer clearance needs better empirical backing

### ISSUE THREAD

**Issue by wittjeff** [w3c/wcag#1445]
on behalf of edX

I am concerned that the minimum pointer target reserved space of 44 CSS pixels may in fact be a reasonable one, but we have not yet adequately demonstrated that this size is most appropriate vs. 40 CSS pixels or 50 CSS pixels or any other size. 

I know that some designers and usability researchers have suggested, based on internal usability studies, that 44 CSS pixels is a good minimum size to minimize touch target errors and target acquisition times.  But how relevant are such mainstream usability studies if we intending this to guideline to benefit users who have dexterity impairments?  An unregulated free market will naturally establish some practical minimum pointer target size, because designers want to keep their users happy. If that minimum size leaves some users out, we need to be able to demonstrate that a proposed larger minimum size is the appropriate _minimum_ amount of constraint on market actors to bring those users back to parity of accessibility.

I did some user research on the topic of minimum touch target sizes with prototype hardware in the early '90s.  Some usability study findings that I recall but unfortunately cannot back up with public data:
- Fingers get larger with age.
- The increase in size of fingers with age may be independent of a more general increase in body weight with age.
- The increase in size of fingers with age may also have a gender effect.  A not-small number of middle-age-and-older men (possibly those who do manual tasks for a living for years?) have very large fingers and demonstrated much more difficulty with small touch targets.  I suspect the number of people in this group is probably larger than the number of people who could be diagnosed with dexterity impairments.
- Virtually no one infers that they can roll their finger to move the center of the touch/splat area to more reliably hit a target that is triggered on the up event.

The first counter-argument against the broad applicability of WCAG to websites and mobile apps (particularly as guidance for broad-reaching regulations)  is that the definition of "accessible" is inherently mushy and depends on whom you ask; that it's really a bunch of Best Practices that have been codified by a select group of well-meaning enthusiasts. Without more rigorous data to back it up, the net effect of adding this SC will be to dilute the strength of the whole set of Guidelines.  

If you want to perform the needed user research to back this one up, I'd like to help.

**Comment by patrickhlauke** [comment]
As a side note, I'll just mention - since your bullet points all center around fingers/touch - that the SC (as well as the existing target size AAA one) applies to mouse as well.

**Comment by detlevhfischer** [comment]
Proposed Working Group answer:
Due to several concerns about the proposed SC Pointer Target Spacing, The Mobile Accessibility Task Force has thought again about the best way to express the requirement and currently proposes an alternative Success Criterion:

> Target Size (AA)
The size of the target for pointer inputs is at least 26 by 26 CSS pixels except when:
> - Inline: The target is in a sentence or block of text;
> - User Agent Control: The size of the target is determined by the user agent and is not modified by the author;
> - Essential: A particular presentation of the target is essential to the information being conveyed.

**Comment by alastc** [comment]
Updated draft working group answer:
<hr>
Due to several concerns about the proposed SC Pointer Target Spacing, The Mobile Accessibility Task Force has thought again about the best way to express the requirement and currently proposes alternative Success Criterion text which you can see in this [understanding document](https://raw.githack.com/w3c/wcag/dd1fd32e49e0ea1141dd1b9220780d26207632dc/understanding/22/pointer-target-spacing.html).

It also includes a link to some research that outlines reasonable sizes in general.  Unfortunately this type of criteria is always going to be a somewhat arbitrary line which balances user-need and feasibility (like color contrast). For example, there is research that 100px is a good minimum for people with mobility impairments on touch screen devices. However, using that size could cause issues for other groups, and would have issues with feasibility.

**Comment by alastc** [comment]
The response above was approved by the working group: https://www.w3.org/2021/02/23-ag-minutes.html#item04
